lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js

```diff
@@ -49,6 +49,7 @@ export default compose(
   connect(
     (state: State) => ({
       packagePanels: getPackagePanelsFromState(state),
+      isLoading: state.ongoingPackageRefreshCount !== 0,
```

Comment:
ongoingPackageRefreshCount shouldn't be in the global state

Reply:
I'm reluctant to modify `PackagesReducer`, which is currently a Map<>; I'd have to either replace it by or move it into an object with the `isLoading` member.

Reply:
You should then create a "globalState" attribut in which you put this attribut, it's cleaner

Reply:
Mh... maybe a `loadingState` attribute and reducer?